.github/workflows/docs.yml

```diff
@@ -5,6 +5,7 @@ on:
     paths:
       # Something has to change in doc/ for thos workflow to be run.
       - "doc/**"
+      - "etc/schema.json"
```

Comment:
I wonder if the docs workflow should now fun SV, as changes to the schema could trigger SV failures. Can be done in another PR though.